website/docs/introduction.md

```diff
@@ -10,61 +10,81 @@ This is a **very early and incomplete alpha release** of Docusaurus 2. We are ma
 **You should use this if**
 
 - :white_check_mark: You want to contribute to Docusaurus 2
-- :white_check_mark: You want to give feedback to make sure it suits your needs
+- :white_check_mark: You want to provide feedback to make sure it suits your needs
 - :white_check_mark: You are curious about what's next as a Docusaurus user
 
 **Do not use this if**
 
-- :x: You are expecting a full production-ready solution (<small>try [Docusaurus 1](https://docusaurus.io/) instead</small>)
-- :x: You are not willing to accept the breaking changes/things not working properly as we improve it during alpha period
+- :x: You need a full production-ready solution (<small>try [Docusaurus 1](https://docusaurus.io/) instead</small>)
+- :x: You prefer not to work with potential breaking changes and/or features not yet working properly as we improve it during alpha period
 
-## Introduction
+## The next Docusaurus is coming to town
 
 <img src="https://docusaurus.io/img/slash-introducing.svg" alt="Docusaurus"/>
 
-Docusaurus 1 used to be a documentation site generator. In Docusaurus 2, we rebuilt it from the ground up, allowing for more customizability but preserved the best parts of Docusaurus 1 - easy to get started. Docusaurus 2 is a **performant static site generator** can be used to create most content-driven websites (e.g. Documentation, Blogs, Product Landing and Marketing Pages, etc) extremely fast. Out of the box, we have support for documentation-style and blogging websites.
+Docusaurus 1 used to be a pure documentation site generator. In Docusaurus 2, we rebuilt it from the ground up, allowing for more customizability but preserved the best parts of Docusaurus 1 - easy to get started, versioned docs, and i18n.
 
-This website for example, is built with Docusaurus 2.
+Beyond that, Docusaurus 2 is a **performant static site generator** and can be used to create most content-driven websites (e.g. Documentation, Blogs, Product Landing and Marketing Pages, etc) extremely fast.
 
-## Features
+While our main focus will still be helping you get your documentations right and well, the shared nature of building this static site generator makes it possible for you to build your whole website interface one-stop.
 
-- ⚛️ **Built using React** - Extend or customize with React
-- 📦 **Uses webpack** - Build an optimized bundle and use any CSS-in-JS solution you like
-- ⚡️ **Lightning Fast** - Link prefetching, instant navigation and page views
-- ✂️ **Easy** - Hot reloading out-of-the-box, automatic route-based code and data splitting! Create new routes by creating new Markdown or JSX files
-- 💥 **Single-page Application** - Client-side navigations and easily add dynamic interactivity to your website (Client-side Rendering)
-- 🎯 **SEO Friendly** - `index.html` files are statically generated for every possible path (Static Rendering)
-- 😌 **Painless** - Painless project setup. First class support for documentation
```

Comment:
why is the comparison removed ? also i think the feature as bullet points are more appealing too.

Reply:
Yeah I think the comparison should stay. People won't need to spend the effort to figure out what the differences are.

I'm neutral about the format of features.

Reply:
with the comparison docs, the docs will be too long. i hope the introduction docs itself is not too long.

Reply:
OK I will bring back the comparisons. I am not against it just that I prefer not to compare in a way that says how others don't fit. So I'll rephrase part of those.

Reply:
yup at least keep the comparisons.

Reply:
> I prefer not to compare in a way that says how others don't fit.

Our intention is to state facts and what each tool is good for. The intention is not to criticize. As a user I'll appreciate fair and accurate comparisons that saves me time from doing it myself.

> I'll rephrase part of those

Yes please 😄